volatility3/framework/symbols/windows/extensions/callbacks.py

```diff
@@ -14,7 +15,7 @@ class _SHUTDOWN_PACKET(objects.StructType, pool.ExecutiveObject):
     It exposes a function which sanity-checks structure members.
     """
 
-    def is_valid(self) -> bool:
```

Comment:
Did you get rid of `is_valid`?  That's also in the API, so would need a version bump to remove them?  I'd recommend `is_parsable` calls `is_valid` to replicate whatever it already did, and then just adds in the check based on the result at the end...

Reply:
Done - I was torn on whether or not to duplicate the device object check in both methods an opted not to to avoid doing the pointer dereference twice, even if it makes the `is_valid` check somewhat less effective.